File: Linguagem C/if_else_lista/Ex_002.c
//2) Faça um programa que receba quatro números e mostre qual deles é o menor.
#include <stdio.h>
int main ()
{
  int num1, num2, num3, num4, menor;

  printf("Digite um valor para num1\n");
  scanf("%d", &num1);

  printf("Digite um valor para num2\n");
  scanf("%d", &num2);

  printf("Digite um valor para num3\n");
  scanf("%d", &num3);

  printf("Digite um valor para num4\n");
  scanf("%d", &num4);

    menor=num1;
  if( num2 < menor ) {
    menor=num2;
  }
  if( num3 < menor ) {
    menor=num3;
  }
  if( num4 < menor ) {
    menor=num4;
  }
  printf("O menor numero sera igual a: %d\n", menor);
  
  return 0;
  
}

File: Python/004_Lista_if_else/Ex_002.py
"""
2) Faça um programa que receba quatro números e mostre qual deles é o menor.
"""
n1 = int(input())
n2 = int(input())
n3 = int(input())
n4 = int(input())

numeros = [n1,n2,n3,n4]
numeros_ord = sorted(numeros)
print(numeros[0])
